provider/infoblox/infoblox.go

```diff
@@ -216,13 +216,8 @@ func (p *InfobloxProvider) Records(ctx context.Context) (endpoints []*endpoint.E
 		}
 		for _, res := range resH {
 			for _, ip := range res.Ipv4Addrs {
-				// host record is an abstraction in infoblox that combines A and PTR records
-				// for any host record we already should have a PTR record in infoblox, so mark it as created
-				newEndpoint := endpoint.NewEndpoint(res.Name, endpoint.RecordTypeA, ip.Ipv4Addr)
-				if p.createPTR {
-					newEndpoint.WithProviderSpecific(providerSpecificInfobloxPtrRecord, "true")
-				}
-				endpoints = append(endpoints, newEndpoint)
```

Comment:
Hi @H3rman8, taking over the pull request by splitting it up in multiple smaller pull requests.
I have a question above these lines of code and why it is removed. Do you know how this one is addressed?

Reply:
Created multiple pull request for different parts of this pull.
This will make it easier to review.

1. #2500
1. #2496
1. #2502
1. #2501

With kind regards,

Reply:
Hi @BobVanB this is addressed at line 311 (end of the function) for both A and H records.